src/android/build.gradle

```diff
@@ -4,7 +4,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.wultra.android.antimalware:antivirus:0.17.0'
+    implementation 'com.wultra.android.antimalware:antivirus:0.18.1'
```

Comment:
Last pull-request is waiting to be added to 0.18.2, which I can release right after it.

Reply:
Upgraded